pkg/tracing/doc.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+/*
+Package tracing helps with propagation of tracing span through context in the
+system. It does this for operations contained to single node, as well as across
+nodes, by injecting special headers.
+
```

Comment:
Package tracing helps with the propagation of the tracing span through context in the
system. It does this for operations contained to single node, as well as across
nodes, by injecting special headers.